main.go

```diff
@@ -1,12 +1,19 @@
 package main
 
-import "github.com/speakeasy-api/speakeasy/cmd"
-
-var (
-	version      = "0.0.1"
-	artifactArch = "linux_amd64"
```

Comment:
why did we remove these? These get set via build tags when goreleaser builds the binaries

Reply:
so the `version` variable will be set to the actual version of the CLI (it isn't a default) and same with `artifactArch`

Reply:
@ThomasRooney was there a particular reason to remove these vs. overriding them?

Reply:
Nvm, fixed